cms_form_example/controllers/main.py

```diff
@@ -21,6 +21,40 @@ def cms_form(self, main_object=None, **kw):
             model, model_id=main_object and main_object.id, **kw)
 
 
+class PartnerFormFieldset(http.Controller, FormControllerMixin):
```

Comment:
Is there a reason you don't extend from PartnerForm? Would save the Redefinition of cms_form on all of them.

Reply:
yes, `http.Controller` does some magic so that if I extends the same controller twice the last one wins and I screw the behavior of the other ones. I need `form_model_key` to stay independent.
Creating a mixin to inherit is not wort here. Finally, this is an example package: I don't care :stuck_out_tongue_winking_eye:

Reply:
well, not good that i screws up inheritance but I see why you're not doing it here :smile: